double_entry.gemspec

```diff
@@ -17,6 +17,32 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry:
+ - New table `double_entry_line_metadata` has been introduced and is required for
```

Comment:
Might be too hard but is there a way to not hardcode the `double_entry_` prefix since it looks like it depends on some sort of config?